content/en/methods/oauth.md

```diff
@@ -195,6 +207,111 @@ If you provide a token you do not own, or no token at all, the API call will ret
+
+The properties exposed by this endpoint can help you better integrate with the Mastodon API, such as allowing for negotiation of `scopes` across different versions of Mastodon.
+
+{{< hint style="info" >}}
+**Example:** You want to use the `profile` scope, but also want to support older Mastodon servers that don't have that scope and would need `read:accounts` instead. You could discover whether a server supports that scope by making a request this endpoint.
+{{< /hint >}}
+
+**Returns:** JSON as per the above description\
+**OAuth:** Public\
+**Version history:**\
+4.3.0 - added
+
+#### Response
+
+##### 200: OK
+
+```json
+{
+  "issuer": "https://social.example/",
+  "service_documentation": "https://docs.joinmastodon.org/",
+  "authorization_endpoint": "https://social.example/oauth/authorize",
+  "token_endpoint": "https://social.example/oauth/token",
+  "app_registration_endpoint": "https://social.example/api/v1/apps",
+  "revocation_endpoint": "https://social.example/oauth/revoke",
+  "scopes_supported": [
+    "read",
+    "write",
+    "write:accounts",
+    "write:blocks",
+    "write:bookmarks",
+    "write:conversations",
+    "write:favourites",
+    "write:filters",
+    "write:follows",
+    "write:lists",
+    "write:media",
+    "write:mutes",
+    "write:notifications",
+    "write:reports",
+    "write:statuses",
+    "read:accounts",
+    "read:blocks",
+    "read:bookmarks",
+    "read:favourites",
+    "read:filters",
+    "read:follows",
+    "read:lists",
+    "read:mutes",
+    "read:notifications",
+    "read:search",
+    "read:statuses",
+    "follow",
+    "push",
+    "profile",
+    "admin:read",
+    "admin:read:accounts",
+    "admin:read:reports",
+    "admin:read:domain_allows",
+    "admin:read:domain_blocks",
+    "admin:read:ip_blocks",
+    "admin:read:email_domain_blocks",
+    "admin:read:canonical_email_blocks",
+    "admin:write",
+    "admin:write:accounts",
+    "admin:write:reports",
+    "admin:write:domain_allows",
+    "admin:write:domain_blocks",
+    "admin:write:ip_blocks",
+    "admin:write:email_domain_blocks",
+    "admin:write:canonical_email_blocks",
+    "crypto"
+  ],
+  "response_types_supported": ["code"],
+  "response_modes_supported": ["query", "fragment", "form_post"],
+  "grant_types_supported": [
+    "authorization_code",
+    "password",
+    "client_credentials"
+  ],
+  "token_endpoint_auth_methods_supported": [
+    "client_secret_basic",
+    "client_secret_post"
+  ]
+}
+```
+
+##### Older Mastodon Versions – 404: Not Found
+
+On Mastodon versions before 4.3.0, requesting this endpoint will result in a `404 Not Found` error.
+
+This is a good indication that the server is running Mastodon older than 4.3.0 and cannot use features added in 4.3.0 and later versions. Instead, you will need to "guess" what that server supports, instead of discovering supported OAuth 2 endpoints, grant flows & scopes dynamically.
```

Comment:
...or not running mastodon at all? or perhaps some other software supports it, etc. point being: i'd take out the sentence about this being a "good indication"

`Instead of dynamically discovering supported OAuth 2 endpoints, grant flows & scopes, you will need to "guess" what the server supports.`